api/cpp/docs/cmake.md

```diff
@@ -1,56 +1,60 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
 <!-- cSpell: ignore ccmake dslint femtovg -->
 
-# Installing Or Building With CMake
+# Setup Developer Environment
 
-Slint comes with a CMake integration that automates the compilation step of the `.slint` markup language files and
-offers a CMake target for convenient linkage.
+## Prerequisite
```

Comment:
I think we need the plural form here, there are multiple things to do after all.

```suggestion
## Prerequisites
```